install.sh

```diff
@@ -521,12 +523,14 @@ sudo pip install -q -U tqdm
 # The BlueView SDK for the Teledyne imaging sonar
 if [[ ! -z "$BVTSDK_PASSWORD" ]]; then
 	instlog "Decrypting and installing the BlueView SDK"
-	cd $CATKIN_DIR/src
-	curl -s https://raw.githubusercontent.com/uf-mil/installer/master/bvtsdk.tar.gz.enc | \
+	cd $BVTSDK_DIR
+	curl -s https://raw.githubusercontent.com/uf-mil/installer/master/libbvtsdk.tar.gz.enc | \
 	openssl enc -aes-256-cbc -d -pass file:<(echo -n $BVTSDK_PASSWORD) | tar -xpz
-	if [[ ! -d $CATKIN_DIR/src/bvtsdk ]]; then
+	if [[ ! -d $BVTSDK_DIR_COMPLETE ]]; then
 		instwarn "Terminating installation due to incorrect password for the BlueView SDK"
 		exit 1
+	elif [[ "$DOCKER" != "true" ]]; then
+		ln -s $BVTSDK_DIR_COMPLETE $CATKIN_DIR/src/mil_common/drivers/mil_blueview_driver/bvtsdk
```

Comment:
Try to comment this block a bit more, it could be confusing for people not involved in developing the CI process.